scripts/bundlesize.config.json

```diff
@@ -0,0 +1,68 @@
+{
```

Comment:
This isn't a script, why move it in `scripts`? If it was at the root we wouldn't need to specify the path in `package.json`.

Reply:
I didn't want a very big root directory, but I guess that can be fixed after